### PR TITLE
Adapt avatar_room.xml for landscape mode and tablets

### PR DESCRIPTION
**Pull request by vasukiarunachalam**
To make the creator room to adapt for landscape and tablet mode , I made following changes in -
layout/avatar_room.xml
layout-land/avatar_room.xml

For normal phones-
values/dimens.xml

For 7" Tablets -
values-sw600dp/dimens.xml
values-sw600dp-port/dimens.xml

For 10" Tablets -
values-sw720dp/dimens.xml
values-sw720dp-port/dimens.xml

This issue also encountered a button id rename from "newUserButton" to "eyeRight" for maintaining naming convention in AvatarRoom.java

### CODE REVIEW

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 1]
```diff
@@ -41,77 +42,89 @@
...
-        android:id="@+id/faceRight"
+        android:id="@+id/eyeLeft"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignRight="@id/face"
-        android:layout_below="@id/face"
-        android:layout_marginTop="@dimen/face_right_margin_top"
-        android:src="@drawable/ic_keyboard_arrow_right_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_left_white_18dp"
+        android:layout_below="@+id/eyes"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: You should only be using `@id` instead of `@+id` when referring to an already-existing id

**Comment by alma-castant** [comment]
I have been testing it in my phone (5") and tablet (8.9") and it looks a little bit off
Phone:


Tablet:

**Comment by vasukiarunachalam** [comment]
This pull request is old. After changing the background I didn't submit a PR. I showed it in issue list and was waiting for your consent. I will submit it sooner.

**Comment by alma-castant** [comment]
Ok. Let me know when you submit the PR :)

**Pull request closed without merging**